src/AddressBook/ABMultiValue.cs

```diff
@@ -47,19 +47,19 @@
 
 namespace AddressBook {
 
-#if !NET
-	[Deprecated (PlatformName.iOS, 9,0)]
-	[Introduced (PlatformName.MacCatalyst, 14, 0)]
-	[Deprecated (PlatformName.MacCatalyst, 14, 0, message : "Use the 'Contacts' API instead.")]
-#else
-	[UnsupportedOSPlatform ("ios9.0")]
-	[UnsupportedOSPlatform ("maccatalyst14.0")]
+#if NET
 	[SupportedOSPlatform ("maccatalyst14.0")]
+	[UnsupportedOSPlatform ("maccatalyst14.0")]
+	[UnsupportedOSPlatform ("ios9.0")]
 #if __MACCATALYST__
-	[Obsolete ("Starting with maccatalyst14.0 use the 'Contacts' API instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+	[Obsolete ("Starting with maccatalyst14.0 Use the 'Contacts' API instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```

Comment:
The capitalization here is not correct English.

Reply:
Ok, I'll teach mellite to lower case when gluing them together.

Reply:
Corrected in this and other active PRs - https://github.com/chamons/mellite/commit/1c4b860045fb3e3575948e65cdd5236c67f62fc3